src/source/Response.h

```diff
@@ -13,12 +13,11 @@ extern "C" {
 // Setting this timeout to terminate CURL connection
 #define TIMEOUT_AFTER_STREAM_STOPPED (1 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND)
 
+#define TIMEOUT_TEARDOWN_CURL_NO_DATA (60ULL * HUNDREDS_OF_NANOS_IN_A_SECOND)

```

Comment:
We can use 30 seconds as this is what we have in curl configuration - 30 bytes in 30 seconds. We could use the same definition here